modules/core_project_factory/scripts/preconditions/preconditions.py

```diff
@@ -405,10 +405,13 @@ def validators_for(opts, seed_project):
     to check and generate corresponding validators.
     """
     validators = [
-        SeedProjectServices(seed_project),
         BillingAccount(opts.billing_account),
```

Comment:
I believe ordering is important here, because you can reach a bad state if the seed project doesn't have required APIs but we try to validate the billing account.

Could you make sure BillingAccount validators get appended after the SeedProject?

Otherwise LGTM.